### PR TITLE
manifests: add resource requests to downloads pod

### PR DESCRIPTION
**Pull request by sjenning**
in service to https://github.com/openshift/origin/pull/22787

### CODE REVIEW

File: manifests/07-downloads-deployment.yaml [hunk 1]
```diff
@@ -38,6 +38,10 @@ spec:
           name: http
           protocol: TCP
         command: ["/bin/sh"]
+        resources:
+          requests:
+            cpu: 10m
+            memory: 50Mi
```
Comment (spadgett) [inline manifests/07-downloads-deployment.yaml]: This seems too low for what the pod actually uses :(

Comment (spadgett) [reply manifests/07-downloads-deployment.yaml]: Screen Shot 2019-05-16 at 4 39 34 PM

Comment (sjenning) [reply manifests/07-downloads-deployment.yaml]: much of that is fs cache.  i suspected that and did a `drop_caches` on the worker running the pod and it dropped to 30Mi.

**Pull request merged**